README.md

```diff
@@ -72,16 +73,16 @@ stepsToRunConfigPath=src/main/resources/stepsToRun.config
 ```
 <b> Running the program </b>
 
-Now that the Pathway-Exchange project is accessible and the `config.properties` file set, the step can be run using the script runner <a href="https://github.com/reactome/data-release-pipeline/blob/feature/download-directory/downloadDirectory/runDownloadDirectory.sh">runDownloadDirectory.sh</a>.
+Now that the Pathway-Exchange project is accessible and the `config.properties` file set, the step can be run using the script runner <a href="https://github.com/reactome/release-download-directory/blob/develop/runDownloadDirectory.sh">runDownloadDirectory.sh</a>.
 
-<b>Note</b>: The <b>ReactomeBook</b> and <b>CreateReleaseTarball</b> steps still use the old Perl scripts found <a href="https://github.com/reactome/Release/tree/master/scripts/release/download_directory">here</a>. This means that, to generate the their output files, Download Directory will need to be run in either a docker container with the <a href="https://github.com/reactome/Release">Release</a> project usable (see <a href="https://github.com/reactome/Release/tree/release-container/container">release-container</a> project), or on the release server. Download Directory can be run anywhere despite this, but if these steps aren't commented out in <a href="https://github.com/reactome/data-release-pipeline/blob/feature/download-directory/downloadDirectory/src/main/resources/stepsToRun.config">stepsToRun.config</a> (see below), they will report errors.
+<b>Note</b>: The <b>ReactomeBook</b> and <b>CreateReleaseTarball</b> steps still use the old Perl scripts found <a href="https://github.com/reactome/Release/tree/master/scripts/release/download_directory">here</a>. This means that, to generate the their output files, Download Directory will need to be run in either a docker container with the <a href="https://github.com/reactome/Release">Release</a> project usable (see <a href="https://github.com/reactome/Release/tree/release-container/container">release-container</a> project), or on the release server. Download Directory can be run anywhere despite this, but if these steps aren't commented out in <a href="https://github.com/reactome/release-download-directory/blob/develop/src/main/resources/stepsToRun.config">stepsToRun.config</a> (see below), they will report errors.
```

Comment:
These points can actually be removed. The release tarball is hasn't been done for over a year, and the Book is now done via Event-pdf project.

Reply:
Nevermind, I didn't do a good job reading markup, this is a note about what they do despite them being legacy steps

Reply:
I just added a little bit of info to make it clearer that ReactomeBook is replaced and release tarball is retired: https://github.com/reactome/release-download-directory/pull/2/commits/f5a3fd6a57b2695ca686c5b774da1a6ef40151f9